Examples/MAX78000/CNN/actionrecognition-demo/softmax.c

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (C) 2010-2018 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* 
+ * Portions Copyright (C) 2020 Maxim Integrated Products, Inc.
+ */
```

Comment:
This has to be updated to:
```
/*
 * Copyright (C) 2010-2018 Arm Limited or its affiliates. All rights reserved.
 * 
 * Portions Copyright (C) 2020 Maxim Integrated Products, Inc. (now owned 
 * by Analog Devices, Inc.).
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 * Licensed under the Apache License, Version 2.0 (the License); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an AS IS BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

However, if we've edited this file within the past few years or even this year, then the copyright will have to be:
```
/*
 * Copyright (C) 2010-2018 Arm Limited or its affiliates. All rights reserved.
 * 
 * Portions Copyright (C) 2020-202x Analog Devices, Inc.
 *
 * SPDX-License-Identifier: Apache-2.0
 *
 * Licensed under the Apache License, Version 2.0 (the License); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an AS IS BASIS, WITHOUT
 * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Please note, you will have to change `202x` to the appropriate year in line ` * Portions Copyright (C) 2020-202x Analog Devices, Inc.`